code/game/objects/items/cardboard_cutouts.dm

```diff
@@ -168,6 +168,11 @@
 			desc = "A cardboard cutout of a laughter demon."
 			icon = 'icons/mob/mob.dmi'
 			icon_state = "bowmon"
+		if("Private Security Officer")
+			name = "Private Security Officer"
+			desc = "A cardboard cutout of a private security officer."
```

Comment:
"They seem to be ignoring you... Typical"
let's pretend they're not cardboard, but we'll all -know- they are.
you can put the desc on the mapped in only ones if you want.